Descent3/pilot.cpp

```diff
@@ -1023,7 +1023,7 @@ void PilotSelect() {
 
     case UID_CANCEL: {
       // Cancel out
-      bool found_old = (cfexist(old_file) != CFES_NOT_FOUND);
+      bool found_old = (cfexist(old_file) != CFES_NOT_FOUND && *old_file);
```

Comment:
I'd rewrite whole logic to get rid `found_old` variable. On `old_file` initialization we already checked that is exists or not. So instead of using `found_old` we may check `old_file[0] == '\0'`.
We can even change `old_file` to `std::fs::path` and use `.empty()` for checks.

Reply:
you're right, I was lazy on that one :P fixed